devicetypes/smartthings/zooz-multisiren.src/zooz-multisiren.groovy

```diff
@@ -0,0 +1,198 @@
+	capability "Actuator"
+	capability "Alarm"
+	capability "Switch"
+	capability "Health Check"
+	capability "Temperature Measurement"
+	capability "Relative Humidity Measurement"
+	capability "Battery"
+	capability "Tamper Alert"
+	capability "Refresh"
+
+	fingerprint mfr: "027A", prod: "000C", model: "0003", deviceJoinName: "Zooz S2 Multisiren ZSE19"
+
+ }
+
+ tiles(scale: 2) {
+	multiAttributeTile(name:"alarm", type: "generic", width: 6, height: 4) {
+		tileAttribute ("device.alarm", key: "PRIMARY_CONTROL") {
+			attributeState "off", label:'off', action:'alarm.siren', icon:"st.alarm.alarm.alarm", backgroundColor:"#ffffff"
+			attributeState "both", label:'alarm!', action:'alarm.off', icon:"st.alarm.alarm.alarm", backgroundColor:"#e86d13"
+		}
+	}
+	 
+	 valueTile("temperature", "device.temperature", inactiveLabel: false, width: 2, height: 2) {
+		 state "temperature", label:'${currentValue}°',
+			 backgroundColors:[
+				 [value: 32, color: "#153591"],
+				 [value: 44, color: "#1e9cbb"],
+				 [value: 59, color: "#90d2a7"],
+				 [value: 74, color: "#44b621"],
+				 [value: 84, color: "#f1d801"],
+				 [value: 92, color: "#d04e00"],
+				 [value: 98, color: "#bc2323"]
+			 ]
+	 }
+		
+	valueTile("humidity", "device.humidity", inactiveLabel: false, width: 2, height: 2) {
+		 state "humidity", label:'${currentValue}% humidity', unit:""
+	 }
+	
+	valueTile("battery", "device.battery", inactiveLabel: false, decoration: "flat", width: 2, height: 2) {
+		 state "battery", label:'${currentValue}% battery', unit:""
+	 }
+	
+	standardTile("refresh", "command.refresh", inactiveLabel: false, decoration: "flat", width: 2, height: 2) {
+		state "default", label:'', action:"refresh.refresh", icon:"st.secondary.refresh"
+	}
+
+	valueTile("tamper", "device.tamper", height: 2, width: 2, decoration: "flat") {
+		state "clear", label: 'tamper clear', backgroundColor: "#ffffff"
+		state "detected", label: 'tampered', backgroundColor: "#ff0000"
+	}
+   
+	main "alarm"
+	details(["alarm", "humidity", "battery", "temperature", "tamper", "refresh"])
+	
+	}
+}
+
+def installed() {
+	initialize()
+}
+
+def refresh() {
+	def commands = []
+	 //get temperature value
+	commands << secure(zwave.sensorMultilevelV5.sensorMultilevelGet(sensorType: 0x01))
+	//get humidity value
+	commands << secure(zwave.sensorMultilevelV5.sensorMultilevelGet(sensorType: 0x05))
+	//get tamper state
+	commands << secure(zwave.notificationV3.notificationGet(notificationType: 0x07))
+	//get state of device (on or off)
+	commands << secure(zwave.basicV1.basicGet())
+	//get battery value
+	commands << secure(zwave.batteryV1.batteryGet())
+	
+	commands
+}
+
+def initialize() {
+	sendEvent(name: "checkInterval", value: 2 * 60 * 60 + 2 * 60, displayed: true, data: [protocol: "zwave", hubHardwareId: device.hub.hardwareID, offlinePingable: "1"])
```

Comment:
This device should not be `offlinePingable` imo

Also Kianoosh has asked that we change the checkInterval formula to 2 * checkInPeriod + 10 minutes, not 2 minutes.

Reply:
done